aerosol_dynamics/particle.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from aerosol_dynamics import physical_parameters as pp
+
+class Particle:
```

Comment:
I will wait to judge this specific `class` treatment of particles until later... It may be end up too limiting